packages/taco/test/conditions/lingo.test.ts

```diff
@@ -165,18 +164,21 @@ describe('check large numbers', () => {
       },
     };
 
+    console.log(uint256ContractConditionProps);
+
     const contractCondition = new ContractCondition(
       uint256ContractConditionProps,
     );
     const conditionExpr = new ConditionExpression(contractCondition);
     expect(conditionExpr.toJson()).toContain(
-      '115792089237316195423570985008687907853269984665640564039457584007913129639935',
+      (115792089237316195423570985008687907853269984665640564039457584007913129639935).toString(
+        16,
+      ),
```

Comment:
We would use '1157.....9935' if we adopted converting the large numbers to strings. But I think converting them to hex string is better because it start with `0x`.